AcsEmulator/AcsEmulatorAPI/Models/CallAutomationModels.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace AcsEmulatorAPI.Models
+{
+    public record CreateCallRequest(string CallbackUri);
```

Comment:
I'll approve for now to unblock quickly but FYI the request model has "targets" as the required field also, not just "callbackUri" 😄 

![image](https://github.com/DominikMe/acs-emulator/assets/46465618/bd071e56-05a8-4a76-a9e0-d43418df36c4)